ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/container.html

```diff
@@ -13,9 +13,19 @@
   ~ See the License for the specific language governing permissions and
   ~ limitations under the License.
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+
+<div class="form" data-sly-use.container="com.adobe.cq.forms.core.components.models.form.FormContainer" data-sly-test.isEdgeDeliveryRequest="${container.edgeDeliveryRequest}">
+    <div>
+        <div>
+      <pre>
+          <code>${container.formDefinition}</code>
+      </pre>
+        </div>
+    </div>
+</div>
+<sly data-sly-test="${!isEdgeDeliveryRequest}">
 <sly data-sly-test="${wcmmode.edit}" data-sly-use.allowed="com.day.cq.wcm.foundation.AllowedComponentList"></sly>
-<form data-sly-use.container="com.adobe.cq.forms.core.components.models.form.FormContainer"
-      data-sly-use.grid="com.day.cq.wcm.foundation.model.responsivegrid.ResponsiveGrid"
+<form data-sly-use.grid="com.day.cq.wcm.foundation.model.responsivegrid.ResponsiveGrid"
```

Comment:
now we'll be having a wrapper <div class = "form"> for core components based rendition also, correct?

Reply:
no @nit23uec , the body tag will have a child tag of main which will enclose the entire content of body.
<img width="1728" alt="Screenshot 2024-04-22 at 10 01 48 AM" src="https://github.com/adobe/aem-core-forms-components/assets/16714569/3aff25ad-9508-4ce2-8963-d3069e40b095">